PowerEditor/src/WinControls/FileBrowser/fileBrowser_rc.h

```diff
@@ -31,24 +31,24 @@
 
 #define	IDD_FILEBROWSER		3500
 
-#define IDD_FILEBROWSER_MENU			 (IDD_FILEBROWSER + 10)
+#define IDD_FILEBROWSER_MENU				(IDD_FILEBROWSER + 10)

```

Comment:
All the reformating of code will be rejected. Check coding standard:
https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/CONTRIBUTING.md